wiki/en/FAQ.md

```diff
@@ -28,9 +28,9 @@ Bear in mind also that all participants should follow [The Golden Rule](/wiki/Cl
 
 ### How do I know if I can join a server? Are there rules?
 
-In general, if somebody sets up a public server, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
+In general, if somebody lists a Server on one of the Public Directories provided by Jamulus by default, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some servers may state their policies in the welcome message you will see in the chat window.
 
-You can also set up a private server on Jamulus and give others your address to connect to that. Have a look at [this page for more information](/wiki/Running-a-Server).
+Note also that servers do not have to be registered on a Directory in order for Jamulus clients to connect to them. Server operators can simply give out the address of their  servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-types).
```

Comment:
```suggestion
Note also that servers do not have to be registered on a (public) Directory in order for Jamulus clients to connect to them. Server operators can simply give out the address of their servers to those they want to play with, and they can then connect directly. Have a look at [this overview for more information](/wiki/Running-a-Server#server-types).
```

Reply:
> Not very happy about a radical change like that,...

Yeah I know what you mean and I think it's unlikely we'll stop people from using the terms "private" and "public" for servers. But it is what it is. A Directory can be public or private, so adding "(public") isn't correct here I don't think.

Reply:
`(public)` adds nothing and probably confuses things.  It's got nothing to do with what's being discussed in the paragraph.